includes/tracks/events/class-wc-settings-tracking.php

```diff
@@ -0,0 +1,142 @@
+<?php
+/**
+ * WooCommerce Settings Tracking
+ *
+ * @package WooCommerce\Tracks
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * This class adds actions to track usage of WooCommerce Settings.
+ */
+class WC_Settings_Tracking {
+	/**
+	 * Singleton instance.
+	 *
+	 * @var WC_Settings_Tracking
+	 */
+	protected static $instance = null;
+
+	/**
+	 * Whitelisted WooCommerce settings to potentially track updates for.
+	 *
+	 * @var array
+	 */
+	protected $whitelist = array();
+
+	/**
+	 * WooCommerce settings that have been updated (and will be tracked).
+	 *
+	 * @var array
+	 */
+	protected $updated_options = array();
+
+	/**
+	 * Instantiate the singleton.
+	 *
+	 * @return WC_Settings_Tracking
```

Comment:
Missing description.

Reply:
Weird, the linter didn't complain about that.

Reply:
Fixed in d1927fb05.